.npmrc

```diff
@@ -0,0 +1 @@
+@fjuul:registry=https://npm.pkg.github.com/fjuul
```

Comment:
I think the npmrc isn't actually needed because we aren't including packages from npm.pkg.github.com here (and publishing will use the registry as defined in the package.json)